apps/deploy-web/src/store/networkStore.ts

```diff
@@ -66,8 +69,42 @@ export const initiateNetworkData = async () => {
   );
 };
 
-const selectedNetwork = atom<Network>(networks[0]);
+const selectedNetworkId = atomWithStorage<Network["id"]>(
+  "selectedNetworkId",
+  defaultNetworkId,
+  // TODO: remove once we have all clients using this store instead of the localstorage
+  //   Issue: https://github.com/akash-network/console/issues/297
+  typeof window !== "undefined"
+    ? {
+        setItem: (key, newValue) => {
+          localStorage.setItem(key, JSON.stringify(newValue));
+          location.reload();
+        },
+        getItem: key => {
+          const raw = localStorage.getItem(key) as Network["id"];
+          return raw ? JSON.parse(raw) : null;
+        },
+        removeItem: key => {
+          localStorage.removeItem(key);
+          location.reload();
+        }
+      }
+    : undefined,
+  { getOnInit: true }
+);
+const selectedNetwork = atom<Network, [Network], void>(
+  get => {
+    const networkId = get(selectedNetworkId);
+    return networks.find(n => n.id === networkId) ?? networks[0];
+  },
+  (get, set, next) => {
+    set(selectedNetworkId, next.id);
+  }
+);
 
 export default {
-  selectedNetwork
+  selectedNetworkId,
+  selectedNetwork,
+  getSelectedNetwork: () => store.get(selectedNetwork),
+  useSelectedNetwork: () => useAtom(selectedNetwork)[0]
```

Comment:
Should we could return the atom instead of just the value so we can set as well?

Reply:
there is an atom above that can be used. This ones if for backward compatibility with the prev implementation. I can add this detail to the task